docs/getting-started/release-notes/v2_6_0.md

```diff
@@ -99,9 +89,29 @@ Zowe Version 2.6.0 contains the bug fixes that are described in the following to
 
 #### z/OS FTP Plug-in for Zowe CLI
 
-- Throw error when list jobs with invalid prefix or owner.
+- Updated the `list jobs` command to throw an error when an invalid prefix or owner is specified. [(Zowe Explorer #1971)](https://github.com/zowe/vscode-extension-for-zowe/issues/1971)
+
+### Zowe Explorer
+
+- Updated UI/UX method calls to use standalone Gui module for better usability and maintainability. ([#1967](https://github.com/zowe/vscode-extension-for-zowe/issues/1967))
```

Comment:
this entry is duplicated.
it should stay as a bug-fix and removed from the enhancement section as it does not introduce new behavior.